src/languages/es.js

```diff
@@ -399,16 +400,16 @@ export default {
         growlMessageOnSave: 'Su tarteja de débito se agregó correctamente',
         expensifyPassword: 'Contraseña de Expensify',
         error: {
-            invalidName: 'Por favor ingrese un nombre válido',
-            addressZipCode: 'Por favor ingrese un código postal válido',
-            debitCardNumber: 'Ingrese un número de tarjeta de débito válido',
+            invalidName: 'Por favor ingresa un nombre válido',
+            addressZipCode: 'Por favor ingresa un código postal válido',
+            debitCardNumber: 'Ingresa un número de tarjeta de débito válido',
             expirationDate: 'Por favor introduzca una fecha de vencimiento válida',
-            securityCode: 'Ingrese un código de seguridad válido',
```

Comment:
This change is not related to this PR but added since it was asked [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1667921807574749?thread_ts=1667921293.250459&cid=C01GTK53T8Q)